cmd/jwt/app.go

```diff
@@ -280,3 +291,7 @@ func isEs() bool {
 func isRs() bool {
 	return strings.HasPrefix(*flagAlg, "RS") || strings.HasPrefix(*flagAlg, "PS")
 }
+
+func isEd() bool {
+	return strings.HasPrefix(*flagAlg, "ED")
```

Comment:
This does not accept `ed25519`, only `ED25519`. The former seems the common way to write it though.
```suggestion
	return strings.HasPrefix(strings.ToUpper(*flagAlg), "ED")
```